website/docusaurus.config.js

```diff
@@ -82,6 +82,15 @@ const config = {
   // - force trailing slashes for deploy previews
   // - avoid trailing slashes in prod
   trailingSlash: isDeployPreview,
+  headTags: [
+    {
+      tagName: "link", 
+      attributes: {
+        rel: "monetization", 
+        href: "https://ilp.uphold.com/LwQQhXdpwxeJ"
```

Comment:
Yeah - it would be super weird if the Docusaurus website was sending money to my uphold account! It was in place to demonstrate that the mechanism worked. To read more about payment pointers (the URL) then read this blog post: https://blog.logrocket.com/getting-started-web-monetization-api/